packages/driver/src/cy/logGroup.ts

```diff
@@ -27,6 +27,7 @@ export default (Cypress, userOptions: Cypress.LogGroup.Config, fn: Cypress.LogGr
   const endLogGroupCmd = $Command.create({
     name: 'end-logGroup',
     injected: true,
+    args: [],
```

Comment:
Given this is an injected command, why do we need this?

Reply:
Without this, we get errors in `commands/querying/root.cy.js` and `commands/querying/within.cy.js`, saying "args is not iterable." from this line in command_queue.ts:

  `args = [command.get('chainerId'), ...args]`

I could add a guard against commands that don't have args in there, or I could ensure that every command has an args array and not need to guard against it. I prefer consistent object shapes over null checks when possible.

The follow-up PR I'm currently working on removes the concept of 'injected' entirely (while maintaining all existing functionality and without editing any of the related tests).